corehq/apps/reports/dbaccessors.py

```diff
@@ -12,4 +13,5 @@ def get_exports(domain):
         startkey=startkey,
         endkey=endkey,
         include_docs=True,
+        stale=settings.COUCH_STALE_QUERY,
     )
```

Comment:
(that plus `skippable_quickcache` would also make it not hit the db at all if it's been loaded recently, but in this case that may not actually get us much)